inc/log.class.php

```diff
@@ -679,7 +679,7 @@ static function getHistoryData(CommonDBTM $item, $start = 0, $limit = 0, array $
                   }
                   $tmp['change'] = sprintf(__('%1$s: %2$s'),
                                            $action_label,
-                                           sprintf(__('%1$s (%2$s)'), $tmp['field'], $data["old_value"]));
+                                           sprintf(__('%1$s (%2$s)'), $tmp['field'], $data["new_value"]));
```

Comment:
Is this normal that the deleted id is stored in `new_value` ?

Reply:
As far as I can see, new/old is kind of inverted for subitems (looking at other possible cases)